Source/ThirdParty/unifdef/README.WebKit

```diff
@@ -0,0 +1,5 @@
+This directory contains unifdef 2.12 downloaded from:
+https://dotat.at/prog/unifdef/unifdef-2.12.tar.gz
+
+When updating to a new release please keep this README.WebKit (updated) and CMakeLists.txt files
+intact.
```

Comment:
I am having trouble parsing this... Are they to be updated or to be kept intact? How about “keep this README.WebKit and CMakeLists.txt files in place, updated as needed”?